app/src/main/java/org/sopt/and/feature/signup/SignUpScreen.kt

```diff
@@ -0,0 +1,260 @@
+                painter = painterResource(R.drawable.ic_info_24),
+                contentDescription = stringResource(R.string.img_password),
+                tint = Color.Gray,
+            )
+            Text(
+                text = stringResource(R.string.sign_up_password_description),
+                fontSize = 12.sp,
+                color = Color.Gray,
+                modifier = Modifier
+                    .padding(start = 3.dp, end = 10.dp),
+            )
+        }
+
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier
+                .padding(top = 50.dp),
+        ) {
+            Box(
+                modifier = Modifier
+                    .height(1.dp)
+                    .weight(1f)
+                    .background(Color.DarkGray),
+            )
+            Text(
+                text = stringResource(R.string.sign_up_social_description),
+                fontSize = 12.sp,
+                color = Color.Gray,
+                modifier = Modifier
+                    .padding(horizontal = 10.dp),
+            )
+            Box(
+                modifier = Modifier
+                    .height(1.dp)
+                    .weight(1f)
+                    .background(Color.DarkGray),
+            )
+        }
+
+        SocialLoginButton(
+            modifier = Modifier
+                .padding(top = 20.dp)
+        )
+
+        Text(
+            text = stringResource(R.string.sign_up_information),
+            fontSize = 12.sp,
+            color = Color.Gray,
+            modifier = Modifier
+                .padding(
+                    top = 40.dp,
+                    start = 20.dp,
+                    end = 20.dp
+                ),
+        )
+
+        Spacer(
+            modifier = Modifier.weight(1f)
+        )
+
+        Text(
+            text = stringResource(R.string.sign_up_button),
+            fontSize = 16.sp,
+            color = Color.White,
+            textAlign = TextAlign.Center,
+            modifier = Modifier
+                .background(Color.Gray)
+                .fillMaxWidth()
+                .clickable {
+                    onSignUpButtonClick(email.value, password.value)
+                }
+                .padding(vertical = 10.dp)
+        )
+    }
+}
+
+@Composable
+private fun SignUpTitle(
+    modifier: Modifier,
+    whiteText: String,
+    greyText: String,
+) {
+    Row(
+        modifier = modifier,
+    ) {
+        Text(
+            text = whiteText,
+            fontSize = 20.sp,
+            color = Color.White,
+        )
+        Text(
+            text = greyText,
+            fontSize = 20.sp,
+            color = Color.Gray,
+        )
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun SignInPreview() {
+    ANDANDROIDTheme {
+        SignUpScreen(
+            onSignUpButtonClick = TODO(),
+            modifier = TODO(),
+            onIdChange = TODO(),
+            onPasswordChange = TODO()
```

Comment:
이러면 preview가 동작하나요? (단순 궁금증)

```suggestion
            onSignUpButtonClick = {},
            modifier = {},
            onIdChange = {},
            onPasswordChange = {},
```

(패러미터 두개 이상 있으면 {_, _ -> } 이렇게 해야하지만) 이렇게 하는게 보는 입장에서 좀 더 안정적일 것 같습니다.

Reply:
파라미터가 있을 때 (_, _ -> ) 이렇게 할 수 있는건 처음 알았어요..!! 오호.... 확실히 제가 보기에도 아래 방식대로 하는 것이 더 좋을 것 같네요 감사합니다!!